### PR TITLE
Add subscribe function

### PR DESCRIPTION
**Pull request by academey**

### CODE REVIEW

File: app/components/home/index.tsx [hunk 1]
```diff
@@ -167,6 +172,15 @@ class HomeComponent extends React.PureComponent<IHomeComponentProps, {}> {
 
     return (
       <div className={styles.homeContainer}>
+        <MetaTags>
```
Comment (TylerShin) [inline app/components/home/index.tsx]: Just for your learning, u can do same thing with react-helmet.

**Review by TylerShin — CHANGES_REQUESTED** [review]
I think we should save the status that an user want to subscribe or not on our spreadsheet.
Can u add additional header field on spreadsheet and take boolean value?
we will send email following by that column.

File: app/components/home/index.tsx (hunk 1 above)
Comment (academey) [reply app/components/home/index.tsx]: 똑같은 일을 React-helmet 에서 하는군요 변경하겠습니다

**Comment by academey** [comment]
합의 봄

**Pull request merged**